source/docs/third-party-services/comments.md

```diff
@@ -86,7 +86,7 @@ Hexo has encode URLs by default since version 4.0: [PR 3708](https://github.com/
 Gitalk relies on the md5 value of the page path for tagging, and this may cause issues. It's recommended to upgrade Hexo to 4.0 (or above), NexT to 7.7.0 (or above) to use Gitalk.
 {% endnote %}
 
-### Valine
+### Valine (China)
```

Comment:
NO china in `/docs/third-party-services/comments.md`.

Reply:
![image](https://user-images.githubusercontent.com/38281581/77896438-e1694200-72aa-11ea-9bac-996b21712cb1.png)

I made a change following the example below and the anchor link is working.

https://deploy-preview-90--theme-next.netlify.com/docs/third-party-services/

Reply:
Parentheses need to be represented as "-" at anchor links

Reply:
oo, thanks. u are right.

Reply:
You're welcome.

~~Two Chinese people are chatting here in English, [funny]~~